src/rider/main/kotlin/com/jetbrains/rider/plugins/efcore/cli/api/DbContextCommandFactory.kt

```diff
@@ -32,6 +32,8 @@ class DbContextCommandFactory(private val intellijProject: Project) {
 
             if (!scaffoldAllTables) {
                 tablesList.forEach {
+                    if (it.isEmpty()) return@forEach
+
                     addNamed("--table", it)
                 }
```

Comment:
This will be slower, efficiency-wise, since we'll iterate over the list twice in comparison to just filtering while iterating it.
But I can make this change if you think it's for the best.

Reply:
In terms of "Big-O" notation both actions have linear complexity (`O(n)` vs `O(2*n)`), even on thousands of items they will not differ much.

But readability is affected, since `return@forEach` is very confusing, a person without any Kotlin experience will first think about it as `break` instead of `continue`. So I suggest you to avoid such returns as much as possible.